components/Feature/Services/index.js

```diff
@@ -107,11 +110,16 @@ const Services = ({
 
     const filteredByCategory = filterByCategories(selectedCategories, categorisedResources);
 
+    const filteredOutExcludedWords =
```

Comment:
I think this creates a problem that if you only select an issue(s) in 'Tick all that apply' it will include everything in the tick all that apply list.

Reply:
hm, that's a good point.... I think it should still include everything, because if we want to include the service that helps people with specific needs, doesn't mean that other services don't help people with those needs, it just means that excluded services only help people with those needs (at least that's how I interpreted it)
So in this case I guess ordering would make more sense?

Reply:
It would be fine to include all specific-need-services, in this scenario, if it weighted the selected item much higher. It's likely if they select 'At risk of homelessness' that this is what they want help with.

I might be misunderstanding your question, the issue is if I select only 'At risk of homelessness' and I don't select any categories or enter a search, then it includes services for 'At risk of homelessness', 'M.E.', 'Carers' and they're equally weighted and amongst a lot of other results.